src/assets/scss/reset.scss

```diff
@@ -8,3 +8,8 @@ body {
     font-family: Roboto, sans-serif;
     margin: 0;
 }
+
+a, a:visited, a:hover, a:active {
```

Comment:
This CSS block is affecting the Breadcrumb component and will affect other links on the page.